go.mod

```diff
@@ -82,7 +82,7 @@ require (
 	github.com/juju/version/v2 v2.0.1
 	github.com/juju/viddy v0.0.0-beta5
 	github.com/juju/webbrowser v1.0.0
-	github.com/juju/worker/v3 v3.5.0
+	github.com/juju/worker/v4 v4.0.0-20240112112653-6268855e643c
```

Comment:
This should just be a semantic version without the sha

Reply:
I'll make a release.